### ISSUE TITLE
How do I identify the REF from a VCF

### ISSUE THREAD

**Issue by hyanwong** [pystatgen/sgkit#1170]
When I use cyvcf2, I can access a `variant.REF` value, but I can't find any obvious references to the word "REF" in the Sgkit docs, for looking at a VCF read in with `vcf_to_zarr`.  I assume that the first in the `variant_alleles` array is the vcf REF: is that guaranteed, and if so, can it be documented?

**Comment by tomwhite** [comment]
Yes, it is guaranteed. The format is documented here: https://github.com/pystatgen/vcf-zarr-spec/blob/main/vcf_zarr_spec.md, where it says:

> Note that the `REF` and `ALT` fields are combined into a single Zarr array.

**Comment by hyanwong** [comment]
Thanks. Is it worth noting that in SGkit somewhere too? By my nitpicky reading of the docs, it doesn't say that they are combine with REF first, either (just that they are "combined")?

**Comment by tomwhite** [comment]
Very true! Are you interested in contributing a patch to fix these issues?

**Comment by hyanwong** [comment]
I'm happy to do so.
